sympy/logic/boolalg.py

```diff
@@ -1304,6 +1304,87 @@ def append_symbol(arg, symbols):
             for c in clauses]
 
 
+def term_to_integer(term):
+	"""
+	Constructs an integer from its base 2 digits.
+
+	Examples
+	========
+
+	>>> from sympy.logic.boolalg import term_to_integer
+	>>> term_to_integer([1, 0, 1, 1])
+	11
+	>>> terms = [[0, 0, 0, 1], [0, 0, 1, 1],
+	...			[0, 1, 1, 1], [1, 0, 1, 1], [1, 1, 1, 1]]
+	>>> map(term_to_integer, terms)
+	[1, 3, 7, 11, 15]
+
+	"""
+
+	term = list(term)
+	return int(''.join(map(str, term)), 2)
+
+
+def integer_to_term(i, n_bits=0):
+	"""
+	Gives a list of the base 2 digits in the integer i.
+	If additional n_bits argument is passed,
+	integer_to_term() extends list of digits with zeros.
+
+	Examples
+	========
+
+	>>> from sympy.logic.boolalg import integer_to_term
+	>>> integer_to_term(14)
+	[1, 1, 1, 0]
+	>>> integer_to_term(7, 5)
+	[0, 0, 1, 1, 1]
+	>>> truthtable = [1, 0, 0, 1, 1, 1, 0, 1]
+	>>> minterms = []
+	>>> n_bits = len(truthtable).bit_length() - 1
+	>>> for i, value in enumerate(truthtable):
+	...			if value:
+	...				minterms.append(integer_to_term(i, n_bits))
+	>>> minterms
+	[[0, 0, 0], [0, 1, 1], [1, 0, 0], [1, 0, 1], [1, 1, 1]]
+
+	"""
+	n = i.bit_length() if i else 1
```

Comment:
bit_length is only available since Py 3.1 so we will have to perhaps define a bin function in core.compatibility and use that, e.g.

...
def bin(s):
    return str(s) if s<=1 else bin(s>>1) + str(s&1)
```